ansible/decommission/archive-instance-services.yml

```diff
@@ -12,15 +12,15 @@
       state: stopped
       enabled: False
     with_items:
+    - prometheus-node-exporter
+    - prometheus-omero-py
+    - omero-logmonitor
```

Comment:
Did this do what you wanted it to? I'd have thought that you'd get alerts due to those agents going down whilst the central monitoring server remained up.

Reply:
I have not and it is possible that the order is incorrect. The decommissioning of `prod86` led to a large number of alerts of type

```
*: prod86-omeroreadonly-3 /opt/omero/server/OMERO.server/var/log/master.err:
-! 9/3/20 12:01:09:480 warning: Blitz-0-Ice.ThreadPool.Server-13: dispatch exception:
   identity: session-06b53b7b-eff9-4c32-93e2-a929c4d91f6e/1a6fc957-ed03-413f-9282-5ee8ca4dca41
   facet: 
   operation: destroy
```

Maybe just stop `omero-logmonitor` before stopping the OMERO services?

Reply:
Yes, I think just stopping omero-logmonitor will reduce some of the log alerts, though probably not all since we also have fluentd.

Reply:
:+1: pushed 077d0c8